cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -30,81 +30,74 @@
     </div>
     <div class="block">
         <div class="expandable-prototypes">
-            {% call expandable.render("Default Expandable") %}
-                {{ lipsum(1, false, 10, 225) | safe }}
-            {% endcall %}
-
-            {% call expandable.render("Bordered Expandable", {
-                    'is_bordered': true
-                }) %}
-                {{ lipsum(1, false, 10, 225) | safe }}
-            {% endcall %}
-
-            {% call expandable.render("Midtone Expandable", {
-                    'is_midtone': true
-                }) %}
-                {{ lipsum(1, false, 10, 225) | safe }}
-            {% endcall %}
-
-            {% call expandable.render("Midtone, Bordered Expandable", {
-                    'is_bordered': true,
-                    'is_midtone':  true
-                }) %}
-                {{ lipsum(1, false, 10, 225) | safe }}
-            {% endcall %}
-
-            {% call expandable.render("Midtone, Bordered, Expandable - Expanded", {
-                    'is_bordered': true,
-                    'is_midtone':  true,
-                    'is_expanded': true
-                }) %}
-                {{ lipsum(1, false, 10, 225) | safe }}
-            {% endcall %}
+            {% include 'molecules/expandable.html' with context %}
+
+
+            {% do global_dict.update(
```

Comment:
Trivial: Follow indenting of https://github.com/cfpb/cfgov-refresh/pull/1251/files#diff-62ffa150f85a8922ad5f88037b0a301bR26